gluten-core/src/main/scala/org/apache/gluten/extension/columnar/OffloadSingleNode.scala

```diff
@@ -232,6 +233,38 @@ case class OffloadFilter() extends OffloadSingleNode with LogLevelUtil {
   }
 }
 
+case class OffloadWindow() extends OffloadSingleNode with LogLevelUtil {
```

Comment:
The optimization should only be done for sort window. Are we missing a condition somewhere?